cli/src/utils/utils.ts

```diff
@@ -215,6 +218,7 @@ export const setPayer = (key: string) => {
 };
 import { existsSync } from "fs";
 import { config } from "dotenv";
+import { compileFunction } from "vm";
```

Comment:
where is it used?

Reply:
it's not must be an auto import good catch

Reply:
fixed